about/index.markdown

```diff
@@ -75,6 +75,10 @@ Please use the following citation for MoveIt!
 
 [PickNik](http://picknik.ai/) was founded with the mission of growing the MoveIt! project through community building and contract consulting.
 
+<a href="https://acutronicrobotics.com"><img src="/assets/images/sponsors/acutronicrobotics.jpg" height="60" style="margin-top:20px"/></a>
```

Comment:
```suggestion
<a href="https://acutronicrobotics.com"><img src="{{ site.url }}/assets/images/sponsors/acutronicrobotics.jpg" height="60" style="margin-top:20px"/></a>
```
The other image paths are made absolute by prepending them with `{{ site.url }}`. Of course this works as well but I think we should keep the same pattern.

Reply:
This is what I did at first but it wouldn't pass the Jenkins test because the file doesn't exist at that path yet.  I'm happy to accept the suggestion, just know that it won't pass the test unless we want to commit the image separately before the code change.

Reply:
Actually I'm not familiar with the current Jenkins setup. Doesn't it allow to set site.url as empty so it would work? Otherwise I think it's fine since it has been done like this before: https://github.com/ros-planning/moveit.ros.org/pull/161/files

Reply:
Either how Jenkins worked changed or it didn't exist then.  I accepted this change so you can see how it's failing.

Reply:
@tylerjw I think this is fine and the resolved url is correct! However, this is a common issue but we could maybe use `url_swap` introduced here https://github.com/gjtorikian/html-proofer/issues/170. This would also help to prevent regressions where image paths become invalid but are still reachable in the current version.

Reply:
yes, let's fix html proofer, from that linked issue they suggest something like:

```
htmlproofer _site --url-swap "https?\:\/\/(localhost\:4000|example\.com):"
```

Reply:
Exactly! I don't think that we need the localhost part but we should be good to go with
```
htmlproofer _site --url-swap "https?\:\/\/moveit\.ros\.org:"
```

Reply:
I tried making this change... (see my below fixup commit) and it still fails.  Any idea what I'm doing wrong?

Reply:
I tried getting this working and I think I got the url-swap to work but now it's complaining about missing alt tags.  I had to modify it slightly to (there are a bunch of non https references and for somereason we have // all over the place and it was bothering me):

`htmlproofer _site --url-swap "http?s?\:\/\/moveit\.ros\.org\/:"`

Reply:
I rebased without these changes and force-pushed so we can deal with on a different PR.